### ISSUE TITLE
Improve README and sample metadata manifest included in download

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/refinebio#333]
### Context and idea

Related to #221. Improving the README and sample metadata manifest came up during the tech team meeting this morning. This has also been noted elsewhere.

From #303:

> ## TODOs
> Improve metadata manifest information
> Improve README

@Miserlou mentioned including sample code for loading files.

### Solution or next step

@Miserlou please fill in details as necessary. 

### New Issue Checklist

- [x] The title is short and descriptive.
- [x] You have explained the context that led you to write this issue.
- [ ] You have reported a problem or idea.
- [ ] You have proposed a solution or next step.

**Comment by Miserlou** [comment]
To clarify:

The current result is a zip of files and a metadata file which looks like (for empty test data):

```
{
    "aggregate_by": "EXPERIMENT",
    "created_at": "2018-07-02T18:29:31",
    "experiments": {
        "GSE51081": {
            "accession_code": "GSE51081",
            "description": "",
            "has_publication": false,
            "protocol_description": "",
            "publication_doi": "",
            "publication_title": "",
            "pubmed_id": "",
            "source_first_published": "",
            "source_last_modified": "",
            "submitter_institution": "",
            "technology": "",
            "title": ""
        }
    },
    "files": [
        "GSE51081.tsv",
        "metadata.tsv",
        "README.md",
        "LICENSE.TXT"
    ],
    "num_experiments": 1,
    "num_samples": 1,
    "samples": {
        "GSM1237810": {
            "accession_code": "GSM1237810",
            "age": "",
            "cell_line": "",
            "compound": "",
            "disease": "",
            "disease_stage": "",
            "genotype": "",
            "organism": "HOMO_SAPIENS",
            "race": "",
            "sample_id": "GSM1237810",
            "sex": "",
            "source_archive_url": "",
            "specimen_part": "",
            "subject": "",
            "time": "",
            "title": "GSM1237810",
            "treatment": ""
        },
        "GSM1237812": {
            "accession_code": "GSM1237812",
            "age": "",
            "cell_line": "",
            "compound": "",
            "disease": "",
            "disease_stage": "",
            "genotype": "",
            "organism": "HOMO_SAPIENS",
            "race": "",
            "sample_id": "GSM1237812",
            "sex": "",
            "source_archive_url": "",
            "specimen_part": "",
            "subject": "",
            "time": "",
            "title": "GSM1237812",
            "treatment": ""
        }
    },
    "scale_by": "STANDARD"
}
```

which is slightly off what was described in #221. The big difference is that we don't have any logic for doing different manifests for different smash types yet (if that's still a good idea). It defaults to "ALL", and just includes all of the files as a list and all of the metadata on a per-experiment and per-sample basis. There are also a few fields missing, like `total_size` and the `sample_experiment_mapping.tsv`, which is a weird one that I don't remember why is listed there. (Is there any need for everything to be directory-ized, since `Homo_sapiens.tsv` and `Gallus_gallus.tsv` can live side by side, then the metadata can all be extracted from the same file?)

The big thing I want is a really easy way to get a dataset loaded into R/pandas/keras DataFrames out of the box and then do some simple example processing. This should either be with a templated code block (README or script file), a link to a really really excellent documentation page via the README file, or a client library.

**Comment by jaclyn-taroni** [comment]
Closed by #640
